metaflow/plugins/argo/argo_workflows_deployer.py

```diff
@@ -240,6 +193,66 @@ def status(self) -> Optional[str]:
 
 class ArgoWorkflowsDeployedFlow(DeployedFlow):
 
+    @classmethod
+    def from_deployment(
+        cls, identifier: str, metadata: Optional[str] = None, _="argo-workflows"
```

Comment:
do we really need the third argument i.e. `_="argo-workflows"` here?

Reply:
Not really -- I added it to keep consistency with from_deployment in DeployedFlow.